langchain/document_loaders/acreom.py

```diff
@@ -0,0 +1,72 @@
+"""Loader that loads acreom vault from a directory."""
+
+import re
+import string
+from pathlib import Path
+from typing import List
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+
+class acreomLoader(BaseLoader):
```

Comment:
nit: naming should be `AcreomLoader` following conventions

Reply:
fixed